test/fixtures/templates.yml

```diff
@@ -38,6 +38,14 @@ pxekickstart:
     locked: false
     type: ProvisioningTemplate
 
+pxeautoyast:
+    name: suse_pxelinux
+    template: DEFAULT linux~LABEL linux~KERNEL <%%= @kernel %>~APPEND initrd=<%%= @initrd %> ramdisk_size=65536 install=<%%= @mediapath %> autoyast=<%%= foreman_url('provision') %> textmode=1
```

Comment:
Just to doublecheck - `<%%=` should be closed with `%%>` but you keep this for consistency with the rest of the file, do you? I believe this works in Ruby for some reason.